htdocs/admin/index.php

```diff
@@ -66,6 +68,7 @@
           <td>
             <a href="edit_challenge.php?id=', htmlspecialchars($challenge['id']), '" class="btn btn-xs btn-primary">Edit</a>
             <a href="new_hint.php?id=', htmlspecialchars($challenge['id']), '" class="btn btn-xs btn-warning">Hint</a>
+            <img src="/img/', ((int)$challenge['expose'] ==1 ? 'accept.png" alt="Exposed"' : 'stop.png" alt="Not Exposed"'), ' />
```

Comment:
I added and retested the requested change, looks to work well. 

changed
    (int)$challenge['expose'] ==1

    to
     (int)$challenge['expose']